script.js

```diff
@@ -1,4 +1,447 @@
+// ____________________  Global Variables ________________________
+
+// L1 Game Mode variables to choose which game user wants to play
+var L1_PICK_GAME_MODE = "Choose your game";
+var L1_NORMAL_MODE = "Normal Mode";
+var L1_LOWEST_MODE = "Lowest Win";
+var L1_AUTO_MODE = "Auto";
+var L1_VARIABLE_DICE_MODE = "Variable Dice";
+var L1_VARIABLE_PLAYERS_MODE = "Variable Players";
+
+// modes to faciliate running of games
+var L2_ROLL_DICE = "Roll Dice Mode";
+var L2_ROLL_DICE_AUTO = "Roll and Auto Pick";
+var L2_CHOOSE_ORDER = "Choose Order Mode";
+var L2_DECIDE_WINNER = "Decide Winner";
+var L2_CHOOSE_NUM_DICE = "Choose number of dice";
+
+// setting initial set up
+var arrayRolls = [];
+var playerOneNum = 0;
+var playerTwoNum = 0;
+var arrayForNums = [];
+var currentPlayer = 1;
+var numOfPlayers = 2;
+var numofDice = 2;
+var L2GameMode = L2_ROLL_DICE;
+var L1GameMode = L1_PICK_GAME_MODE;
+var playerOneScore = 0;
+var playerTwoScore = 0;
+
+// ____________________________SUPPORTING FUNCTIONS_______________________________
+
+// 2 functions for player to select their order
+var selectAB = function () {
+  number = Number(`${arrayRolls[0]}` + `${arrayRolls[1]}`);
+  return number;
+};
+var selectBA = function () {
+  number = Number(`${arrayRolls[0]}` + `${arrayRolls[1]}`);
+  return number;
+};
```

Comment:
Excellent. The naming for functions are great. As well as the code within the function.

Is number here meant to be declared with a `var`? 

If so, perhaps you could have just returned directly, something like - 
```
var selectBA = function () {
  return Number(`${arrayRolls[0]}` + `${arrayRolls[1]}`);
};
```

Reply:
Just a thought, would it be possible to not have a`selectAB` function? Since the default roll is always AB? 
`selectBA()` function is necessary